notebooks/stable-video-diffusion/stable-video-diffusion.ipynb

```diff
@@ -9,7 +9,7 @@
     "# Image to Video Generation with Stable Video Diffusion\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    %pip install -q datasets nncf
</pre><p>please specify lower bound for nncf</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>